apps/src/templates/progress/NewProgressBubble.jsx

```diff
@@ -101,31 +127,49 @@ const NewProgressBubble = React.createClass({
 
     const tooltipId = _.uniqueId();
 
+    // Outer div here is used to make sure our bubbles all take up equivalent
+    // amounts of space, whether they're diamonds or circles
     let bubble = (
       <div
-        style={style}
-        data-tip data-for={tooltipId}
-        aria-describedby={tooltipId}
+        style={{
+          display: 'inline-block',
+          width: (smallBubble ? SMALL_DOT_SIZE : DOT_SIZE) +
+            2 * styles.main.borderWidth + 2 * 2,
```

Comment:
I was trying to make this less magical (it's essentially meant to be 2px of margin on each side). Without a comment though, it might not be any less magical. Will address in a future PR